### DISCUSSION TITLE
Nightscout Follower as local data source provided by Juggluco / poll time

### DISCUSSION THREAD

**Discussion by source2k** [NightscoutFoundation/xDrip#2946] [Q&A]
I'm using Juggluco to read glucose values from Libre 3 sensors.
Juggluco provides a new reading every minute.
Is it possible to increase the Nightscout Follower poll time from 5 minutes to 1 minute?

**Comment by source2k** [comment]
I implemented my suggestion for myself by changing the hard coded time value. Though, a possibility to manually specify the poll time would be interesting for some other users.

**Reply by Teute0815** [reply]
It would be definitely for me. I filed a similar request, but it doesn't seem to get (m)any feedback. #3758 

Could you give an instruction on how to do it myself - if that is at all possible as a solely "frontend user" with no clue in programming?
